plugins/BEdita/API/src/Auth/EndpointAuthorize.php

```diff
@@ -218,7 +218,7 @@ protected function getEndpoint()
             $this->endpoint = $Endpoints->newEntity(
                 [
                     'name' => $endpointName,
-                    'enabled' => true,
+                    'enabled' => 1,
```

Comment:
Why? If `true` does not work it means there's a problem.

Reply:
My mistake, got confused testing on a bad sqlite db where `enabled` was `integer` and not `boolean`
Removing this PR